warehouse/static/js/warehouse/controllers/gitlab_repo_info_controller.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static targets = [
+    "forksCount",
+    "forksUrl",
+    "openIssuesCount",
+    "openIssuesUrl",
+    "stargazersCount",
+    "stargazersUrl",
```

Comment:
"stargazers" seems to be a github-ism, we should just call this "followers" I think.

Reply:
I used "starrers" term here as in the Gitlab's UI and API

Reply:
Ah, perfect, ty!